src/main/java/org/dsa/iot/knx/DeviceFolder.java

```diff
@@ -134,34 +124,50 @@ public Node buildFolderTree(Node parent, Queue<String> path) {
 
 	@Override
 	public void buildDataPoint(Node parent, GroupAddressBean addressBean) {
-		GroupAddress address = null;
+		GroupAddress groupAddress = null;
 		try {
-			address = new GroupAddress(addressBean.getGroupAddress());
+			groupAddress = new GroupAddress(addressBean.getGroupAddress());
 		} catch (KNXFormatException e) {
 			e.printStackTrace();
```

Comment:
If it throws an exception, it'll most likely crash right after. Shouldn't it fail fast instead?